src/sampling.rkt

```diff
@@ -3,9 +3,12 @@
          (only-in fpbench interval range-table-ref condition->range-table [expr? fpcore-expr?]))
 (require "searchreals.rkt" "errors.rkt" "common.rkt"
          "float.rkt" "syntax/types.rkt" "timeline.rkt" "config.rkt"
-         "syntax/sugar.rkt")
+         "syntax/sugar.rkt" "ground-truth.rkt")
```

Comment:
This is a little funny, because I think the old setup is users -> ground-truth -> sampling, and you've kind of inverted it. Anyway, I think it's all fine, more important to keep moving forward here.

Reply:
Perhaps "ground truth" had adopted a different meaning? I view ground truth as our correctly-rounded, round-once evaluator.

Reply:
I think back in the day it was intended to be the specification of what, exactly, it actually meant for a sample to be valid. But that's gotten a bit simpler over time (like inf now being ordinary) so that's become less important.